app/client/components/user/CreateVendorForm.tsx

```diff
@@ -1,120 +0,0 @@
-import React from "react";
```

Comment:
Refactored CreateVendorForm into a single CreateUserForm for simplicity (that's how the API works)